src/app/components/LanguageSources.svelte

```diff
@@ -135,7 +133,13 @@
       </tr>
     {/each}
   </table>
-  <Button color="blue" isOutlined size="large" onClick={handleAddSource}>
+  <Button
+    color="blue"
+    isOutlined
+    size="large"
+    onClick={handleAddSource}
+    data_cy="language-sources-add-source">
     Add Source
   </Button>
+  <AddSource bind:addSource />
```

Comment:
This can be changed in a later PR, but I realize now that the better markup for this is [the `<details>`/`<summary>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details).

Using `<details>`/`<summary>` means you would not even need to have the `addSource` variable to begin with, plus you get accessibility for free!